chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/termination/JTerminationDuration.java

```diff
@@ -0,0 +1,46 @@
+package com.griddynamics.jagger.user.test.configurations.termination;
+
+/**
+ * Limits test execution time.
```

Comment:
Test with such termination strategy will be executed for the defined time - duration